sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_event.hpp

```diff
@@ -111,6 +116,45 @@ struct _ur_ze_event_list_t {
     }
     return *this;
   }
+
+  // This function allows to merge two _ur_ze_event_lists
+  // The ur_ze_event_list "other" is added to the caller list.
+  // Note that new containers are allocated to contains the additional elements.
+  void fusion(const _ur_ze_event_list_t &other) {
```

Comment:
Nitpick but for a slightly more descriptive name, something like `insert` would be more in keeping with standard containers. Only a suggestion though there may be better names.

Also I think the comment here should note that elements are copied here.